### DISCUSSION TITLE
where is the introduction of buildModelsFromUrdf

### DISCUSSION THREAD

**Discussion by LHT-Liu** [stack-of-tasks/pinocchio#2513] [Q&A] [answered]
where is the introduction of buildModelsFromUrdf?

```
model, collision_model, visual_model = pin.buildModelsFromUrdf(urdf_path, mesh_dir, pin.JointModelFreeFlyer())
model: pin.Model = pin.buildModelFromUrdf(urdf_path)
```

I think that the above two model is different,so why?
@jviereck @jcarpent

**Comment by jorisv** [comment, marked as answer]
Hello @LHTLiu,

First one will load `urdf_path`, mesh  root dir should be in `mesh_dir` and the root joint will be a freeflyer.
Second one only load `urdf_path` and the root joint will be fixed.
